src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -25,47 +25,115 @@
+  public static final int SCHEMA_CACHE_SIZE_DEFAULT = 1000;
+
   public static final String STORAGE_CLASS_CONFIG = "storage.class";
   private static final String STORAGE_CLASS_DOC = "The storage layer.";
-  public static final String STORAGE_CLASS_DEFAULT = "io.confluent.connect.hdfs.HdfsStorage";
+  public static final String STORAGE_CLASS_DEFAULT = "io.confluent.connect.hdfs.storage.HdfsStorage";
+
   public static final String RETRY_BACKOFF_CONFIG = "retry.backoff.ms";
   private static final String RETRY_BACKOFF_DOC = "The retry backoff in milliseconds.";
-  public static final long RETRY_BACKOFF_DEFALUT = 5000L;
+  public static final long RETRY_BACKOFF_DEFAULT = 5000L;
+
   public static final String HIVE_HOME_CONFIG = "hive.home";
   private static final String HIVE_HOME_DOC = "Hive home directory";
+  public static final String HIVE_HOME_DEFAULT = "";
+
   public static final String HIVE_CONF_DIR_CONFIG = "hive.conf.dir";
   private static final String HIVE_CONF_DIR_DOC = "Hive configuration directory";
+  public static final String HIVE_CONF_DIR_DEFAULT = "";
+
+  public static final String HIVE_INTEGRATION_CONFIG = "hive.integration";
+  private static final String HIVE_INTEGRAtiON_DOC = "Integrate with Hive";
+  public static final boolean HIVE_INTEGRATION_DEFAULT = false;
+
+  public static final String HIVE_DATABASE_CONFIG = "hive.database";
+  private static final String HIVE_DATABASE_DOC = "The database to use when writing to hive.";
+  private static final String HIVE_DATABASE_DEFAULT = "default";
+
+  public static final String AWAIT_TERMINATION_TIMEOUT_CONFIG = "hive.executor.shutdown.timeout.ms";
+  private static final String AWAIT_TERMINATION_TIMEOUT_DOC = "Hive executor clean shutdown timeout.";
+  private static final long AWAIT_TERMINATION_TIMEOUT_DEFAULT = 3000L;
+
   public static final String SCHEMA_COMPATIBILITY_CONFIG = "schema.compatibility";
   private static final String SCHEMA_COMPATIBILITY_DOC = "The schema compatibility level";
   private static final String SCHEMA_COMPATIBILITY_DEFAULT = "NONE";
 
+  public static final String PARTITIONER_CLASS_CONFIG = "partitioner.class";
+  private static final String PARTITIONER_CLASS_DOC = "The partitioner class.";
+  public static final String PARTITIONER_CLASS_DEFAULT = "io.confluent.connect.hdfs.partitioner.DefaultPartitioner";
+
+  public static final String PARTITION_FIELD_CONFIG = "partition.field";
+  private static final String PARTITION_FIELD_DOC = "The field to partition data.";
+  public static final String PARTITION_FIELD_DEFAULT = "";
+
+  public static final String PARTITION_DURATION_CONFIG = "partition.duration.ms";
+  private static final String PARTITION_DURATION_DOC = "The duration of a partition in time based partitoner.";
+  public static final long PARTITION_DURATION_DEFAULT = -1L;
+
+  public static final String PATH_FORMAT_CONFIG = "path.format";
+  private static final String PATH_FORMAT_DOC = "The time format to use.";
```

Comment:
These docs could give a bit more info, e.g. "The time format to use when partitioning with TimeBasedPartitioner."